app/src/main/java/org/fossasia/openevent/api/processor/SpeakerListResponseProcessor.java

```diff
@@ -42,12 +47,15 @@ public void success(SpeakerResponseList speakerResponseList, Response response)
         dbSingleton.clearDatabase(DbContract.Speakers.TABLE_NAME);
         dbSingleton.insertQueries(queries);
 
+        Bus bus = OpenEventApp.getEventBus();
+        bus.post(new SpeakerDownloadEvent(true));
```

Comment:
All events from processors should be posted to the UI thread and data processing should be on the common task loop